mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/TiltakstypeRoutes.kt

```diff
@@ -46,5 +39,22 @@ fun Route.tiltakstypeRoutes() {
 
             call.respond(tiltakstype)
         }
+
+        put("{id}/tags") {
```

Comment:
Nå som vi får støtte for skriveoperasjoner så må vi vel inn med litt mer restriktiv tilgangskontroll også, Dette endepunktet burde i første omgang kun være tilgjengelige for fagansvarlige?

Reply:
Ja, godt poeng! Skal få sett på det.